agent/src/main/java/com/blackducksoftware/integration/hub/teamcity/agent/scan/HubBuildProcess.java

```diff
@@ -297,6 +306,24 @@ private String getParameter(@NotNull final String parameterName) {
 		return result;
 	}
 
+	private String getConfigParameter(@NotNull final String parameterName) {
+		final String value = context.getConfigParameters().get(parameterName);
```

Comment:
What do you mean? Are you asking do we trimToNull before we pass in the parameterName?